ArduPlane/quadplane.cpp

```diff
@@ -2955,9 +2955,9 @@ void QuadPlane::takeoff_controller(void)
         // start motor spinning if not spinning already so user sees it is armed
         set_desired_spool_state(AP_Motors::DesiredSpoolState::GROUND_IDLE);
         takeoff_start_time_ms = now;
-        if (now - rudder_takeoff_warn_ms > 3000) {
+        if (now - plane.takeoff_state.rudder_takeoff_warn_ms > 3000) {
```

Comment:
I think this 3000 and the other one in takeoff.cpp should be a common const or define, just so we keep them consistent.

Reply:
You could add a const to the `plane.takeoff_state` struct.

Reply:
not sure how to add the constant (ie 3000) to the structure...can you show me? add something like 'uint16_t timeout = 3000; in the structure?

Reply:
Exactly, just need the const. You can also use brace initialisation. 

`const uint16_t timeout {3000};`

Reply:
put it at a define...no need to waste memory